Doc/whatsnew/3.13.rst

```diff
@@ -219,6 +228,12 @@ Deprecated
   They will be removed in Python 3.15.
   (Contributed by Victor Stinner in :gh:`105096`.)
 
+* Passing more than one positional argument to :func:`sqlite3.connect` is
```

Comment:
Why is it duplicated?

Reply:
Deprecations are repeated in the _Deprecated_ section. See prior entries.

Reply:
Let's hear what CAM says; he's in charge of the What's New. cc. @CAM-Gerlach

Reply:
I removed the duplicate entry.